src/main.cpp

```diff
@@ -230,6 +230,11 @@ void haRegisterSensors() {
   haAnnounceSensor(String("temperature"), String("temperature"), false, payload, buffer);
   haAnnounceSensor(String("humidity"), String("humidity"), false, payload, buffer);
 #endif
+#if defined(HAVE_ENS160)
```

Comment:
Based on https://www.home-assistant.io/integrations/sensor/#device-class shouldn't the device classes (i.e. the second arguments to haAnnounceSensor) be:
aqi
volatile_organic_compounds
carbon_dioxide

Reply:
Great catch, lead me to another bug non-standard enums.